app/routes.py

```diff
@@ -153,16 +153,20 @@ def execute_circuit():
     qpu_name = request.json['qpu-name']
     impl_language = request.json.get('impl-language', '')
     impl_url = request.json.get('impl-url')
+    qasm_string = request.json.get('qasm-string', "")
     if type(impl_url) is str:
         impl_url = [impl_url]
     impl_data = request.json.get('impl-data')
     if type(impl_data) is str:
         impl_data = [impl_data]
+    
+
     transpiled_qasm = request.json.get('transpiled-qasm')
     if type(transpiled_qasm) is str:
         transpiled_qasm = [transpiled_qasm]
     bearer_token = request.json.get("bearer-token", "")
     input_params = request.json.get('input-params', "")
+    optimization_level = request.json.get('circuit transpile optimization level', 3)
```

Comment:
Similar to other attributes use dashes "-" in between words.
Also, maybe slightly shorten the attribute name? For example: transpilation-optimization-level